.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: go
 
 go:
-  - 1.4
-  - 1.5
-  - 1.6
-  - tip
+ - 1.5.4
+ - 1.6.4
+ - 1.7.6
+ - 1.8.3
+ - 1.9.0
+ - tip
```

Comment:
Suggestion, use this:

```YML
go:
  - 1.5.x
  - 1.6.x
  - 1.7.x
  - 1.8.x
  - 1.9.x
  - master
```

Reference: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use.

Reply:
Thanks for the feedback, fixed!